### PR TITLE
feat: add hello cmd

### PR DESCRIPTION
**Pull request by marsevilspirit**
支持hello命令

## Summary by CodeRabbit

- **New Features**
	- Introduced a new command "hello" that can be executed without prior authentication.
	- Added a `HelloCmd` class to handle the new command with specific functionalities.
	- Expanded command table to include the new "hello" command.

- **Bug Fixes**
	- Enhanced authentication logic to allow execution of the new command without prior authentication.

- **Chores**
	- Minor formatting change with the addition of a newline in the `client_map.h` file.

### CODE REVIEW

**Comment by marsevilspirit** [comment]
目前支持了基本的hello命令，和`setname`子命令。
还有`auth`子命令还未支持。

**Comment by luky116** [comment]
待 https://github.com/arana-db/kiwi/pull/10 合并后再改进下

File: src/cmd_admin.h [hunk 1]
```diff
@@ -195,6 +196,21 @@ class PingCmd : public BaseCmd {
   void DoCmd(PClient* client) override;
 };
 
+class HelloCmd : public BaseCmd {
```
Comment (AlexStocks) [inline src/cmd_admin.h:199]:
```cpp
class HelloCmd : public BaseCmd {
 public:
  HelloCmd(const std::string& name, int16_t arity);

 protected:
  bool DoInitial(PClient* client) override;

 private:
  void DoCmd(PClient* client) override;

  void Hello(PClient* client);

  private:
      bool authed_ = true;
};

```

Comment (marsevilspirit) [reply src/cmd_admin.h:199]: done

**Review by AlexStocks — APPROVED** [review]

**Review by lqxhub — APPROVED** [review]

**Pull request merged**